tests/format/angular/interpolation/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -100,22 +100,22 @@ trailingComma: "none"
   a[b | c],
   ($students | async).items,
   ($students | async)(),
-  myData | myPipe : "arg1" : "arg2" : "arg3",
+  myData | myPipe: "arg1" : "arg2" : "arg3",
   value
     | pipeA
       : {
```

Comment:
Why not this for consistency?
```suggestion
    | pipeA:
      {
```

Reply:
I just rolled back how prettier treated the first semicolon. I guess this is how it used to work.